src/Seaborn.jl

```diff
@@ -60,7 +63,7 @@ macro delegate(f_list...)
     blocks
 end
 
-@delegate jointplot pairplot kdeplot lmplot regplot residplot interactplot coefplot boxplot violinplot stripplot swarmplot pointplot clustermap tsplot palplot FacetGrid PairGrid JointGrid axes_style set_style plotting_context set_context set_color_codes reset_defaults reset_orig set
+@delegate lineplot scatterplot relplot jointplot pairplot kdeplot lmplot regplot residplot interactplot coefplot boxplot violinplot stripplot swarmplot pointplot clustermap tsplot palplot FacetGrid PairGrid JointGrid axes_style set_style plotting_context set_context set_color_codes reset_defaults reset_orig set
```

Comment:
You don't need to change this line in this case, since you are manually defining those functions below.

Reply:
Okay thanks, I removed `lineplot scatterplot relplot` from the delegate macro in line 66